Added clarification on multi-arch images (#266)
open-component-model/ocm-website: content/docs/tutorials/best-practices-with-ocm.md

toc: true
---

> This chapter contains guidelines for common scenarios how to work with the Open Component Model, focusing on using CI/CD, building, and publishing.

Reviewer: ```suggestion
This chapter contains guidelines for common scenarios how to work with the Open Component Model, focusing on using CI/CD, build and publishing processes.
```

## Building multi-arch images

> **Note:** This section provides information only on on building multi-arch images. If you are instead interested in referencing a multi-arch image, see the [Referencing multi-arch images](#referencing-multi-arch-images) section.

> At the time of writing this guide Docker is not able to build multi-architecture (multi-arch)

Reviewer: ```suggestion
At the time of writing this guide Docker is not able to build multi-architecture (multi-arch / multi-platform)
```

images natively. Instead, the `buildx` plugin is used. However, this implies building and pushing
images in one step to a remote container registry as the local docker image store does not

> support multi-arch images.

Reviewer: ```suggestion
support multi-arch images (also see [this article](https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way)
```
